src/components/BaseButton/BaseButton.vue

```diff
@@ -120,6 +120,7 @@ export default {
 
 <style scoped>
 .base-button {
+  -webkit-appearance: none;
   @apply border-transparent
```

Comment:
There's a tailwind class for that.

```suggestion
  @apply appearance-none
    border-transparent
```

Reply:
Of course there is... 😂 Thanks, @jjandoc!